### ISSUE TITLE
could not get digit-exe-debian-linux-32-bit-installer-10_0.run running on x86_64 ubuntu 16.04

### ISSUE THREAD

**Issue by holla2040** [markummitchell/engauge-digitizer#231]
I fooled around for about an hour installing i386 libs on x64 ubuntu 16.04. 
ldd /usr/local/engauge/engauge.exe
shows all libraries are installed. 

I added /usr/lib/i386-linux-gnu/qt5/plugins/platforms/ to 
/usr/local/engauge/engauge LD_LIBRARY_PATH

Still getting this error

holla@hendrix[566]: /usr/local/engauge/engauge
This application failed to start because it could not find or load the Qt platform plugin "xcb".

Available platform plugins are: xcb.

Reinstalling the application may fix this problem.
/usr/local/engauge/engauge: line 2:  7290 Aborted                 /usr/local/engauge/engauge.exe

Going back to universal version now.

**Comment by markummitchell** [comment]
I set up an ubuntu 16.04 64 bit environment and then loaded 32 bit libraries. Ultimately I got stuck on a library dependency (libdl.so.2:i386) which just cannot be solved. That seems to be the problem with trying to install an executable from one linux distribution on another linux distribution - at some point there is a conflict in the libraries so it does not work. That's why the AppImage releases were added, since they let the application (Engauge in this case) use bundled libraries without ever having to interact with the system libraries. At this point, this issue may be a 'Do not fix'
